InternalDelay.py

```diff
@@ -159,10 +140,18 @@ def _from_SS_lti(P_ss, P_dt):
 
         delays = P_dt
 
-        return A, B1, B2, C1, C2, D11, D12, D21, D22, delays
+        return InternalDelay(A, B1, B2, C1, C2, D11, D12, D21, D22, delays)
 
     @staticmethod
-    def _from_tf_lists(num, den, delays):
+    def from_tf_lists(num, den, delays):
```

Comment:
I'm not sure about this name. Perhaps `from_coefficients` is more descriptive?

Reply:
`from_tf_coefficient_lists` or `from_tf_coefficients`?